pybossa/model/__init__.py

```diff
@@ -68,3 +68,10 @@ def update_project_timestamp(mapper, conn, target):
     sql_query = ("update project set updated='%s' where id=%s" %
                  (make_timestamp(), target.project_id))
     conn.execute(sql_query)
+
```

Comment:
after adding this function, we could save the one that is above (update_project_timestamp) as it is an specific case of this one.